sources/packages/web/src/views/supporting-user/Login.vue

```diff
@@ -8,25 +8,25 @@
               Welcome to StudentAid BC
             </h1>
             <p class="mb-5">
-              Login or sign up to submit supporting information for student
-              applications.
+              We help with the cost of post-secondary education through student
+              loans, grants and other financial assistance programs.
```

Comment:
There is a comma missing in the text. There is a PDF in the ticket with the text that would allow us to copy and paste and avoid this kind of back-and-forth.
The text should be as the below.
```
We help with the cost of post-secondary education through student loans, grants, and other financial assistance programs.
```